### ISSUE TITLE
SendGrid

### ISSUE THREAD

**Issue by IanMcCrabb** [HughCraig/TLCMap#8]
Your Twilio SendGrid account has been temporarily suspended as we have detected that your account's credentials (password and/or API key) are publicly listed on the code repository GitHub. This is a dangerous practice which may result in your account being used by unauthorized third parties to send malicious content and which may incur damage to your reputation as a quality sender and charges against your account for high usage that you did not perform.
Before you ask for your account's reactivation, please ensure that you: 
1) [Change your account's password](https://sendgrid.com/docs/ui/account-and-settings/resetting-your-username-and-password). If your account was created using Heroku or IBM BlueMix, you must use our password reset form.
2) [Delete and update exposed API keys in your account](https://sendgrid.com/docs/ui/account-and-settings/api-keys/#delete-an-api-key)
3) Enable [two-factor](https://sendgrid.com/docs/ui/account-and-settings/two-factor-authentication) authentication for your account
4) Remove your account credentials and API keys from any public listings on code repositories or associated comments on sites such as GitHub or BitBucket.
Please see the following link(s) for locations where your credentials have been found to be publicly exposed:
https://github.com/HughCraig/TLCMap/blob/947d0d8888bbcb42715c968ee0b938c0c80c9421/doc/Gazetteer_Install_Deploy.docx
Please respond directly to this ticket in order to start the reactivation process and be aware that an account in a suspended state at the end of the month will still incur subscription auto-renewal billing. Sending wanted mail is paramount to Twilio SendGrid, therefore we would like to work with you to identify the cause of and resolve this unwanted activity. We apologize for any inconvenience this may have caused and truly appreciate your continued business. Thank you for using Twilio SendGrid.
Sincerely,

**Comment by IanMcCrabb** [comment]
Bill to provide domain and server access for Yang and Ian

**Comment by BillPascoe** [comment]
I fixed this. So many problems with resetting password not working, no 2 factor authentication option, etc etc. I deleted the Twilio SendGrid subscription and recreated it. This involved also:
- re-authenticate the domain in Twilio SG (not individual email address)
- remove the old DNS settings from domain.com and from Azure DNS, and add in the new ones as specified in Twilio SG 
- create a new API Key in SG
- add new API key to the .env file on the server
- test password reset
For future problems I have sent invitations to administer Azure. Let me know if it doesn't work, or if I should have done it differently, such as inviting your already existing Azure accounts or something.
